src/main/java/com/spotify/reaper/storage/CassandraStorage.java

```diff
@@ -631,12 +654,17 @@ public boolean updateRepairSchedule(RepairSchedule newRepairSchedule) {
     Optional<RepairSchedule> repairSchedule = getRepairSchedule(id);
     if(repairSchedule.isPresent()){
       RepairUnit repairUnit = getRepairUnit(repairSchedule.get().getRepairUnitId()).get();
-      BatchStatement batch = new BatchStatement();
-      batch.add(deleteRepairSchedulePrepStmt.bind(repairSchedule.get().getId()));
-      batch.add(deleteRepairScheduleByClusterAndKsPrepStmt.bind(repairUnit.getClusterName(), repairUnit.getKeyspaceName(), repairSchedule.get().getId()));
-      batch.add(deleteRepairScheduleByClusterAndKsPrepStmt.bind(repairUnit.getClusterName(), " ", repairSchedule.get().getId()));
-      batch.add(deleteRepairScheduleByClusterAndKsPrepStmt.bind(" ", repairUnit.getKeyspaceName(), repairSchedule.get().getId()));
-      session.execute(batch);
+
+      session.executeAsync(deleteRepairScheduleByClusterAndKsPrepStmt
```

Comment:
Since we have no automatic retries I figured it would be better to use batches. We only run a few of those so it shouldn't harm the clusters.